doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
-
-For this method to be possible, the IO device need to support Single Root Input Output Virtualization (SR-IOV) which allows it to present itself as multiple devices, known as Physical Functions, PFs, and Virtual Functions, VFs as presented in **Figure 6**.
-
-Each of those Virtual Functions can then be independently assigned exclusively to a workload (with the appropriate hardware support of an IOMMU).
+<a name="2.0"></a>
+# 2. Principles
 
-Similar to the previous method ("Direct Assignment"), this method provides better performance than para-virtualization, but lacks the flexibility and the portability sought and therefore must also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+Any specification work created within CNTT **must** conform to set of principles specified by CNTT:
 
-<p align="center"><img src="./figures/tech_sriov.png" alt="sriov" title="Document Types" width="50%"/></p>
-<p align="center"><b>Figure 6:</b> Device Sharing with SR-IOV & Direct Assignment.</p>
+* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
+* [Reference Architecture Principles](../ref_arch#principles)
 
-<a name="4.1.4"></a>
-#### 4.1.4 Para-Virtualization method (Hardware support)
+<a name="3.0"></a>
+# 3. Scope
 
-This method basically is a mixture between the software only para-virtualization method and the direct assignment method (including the device sharing method) where the frontEnd driver which is running on the workload is a standard off the shelf driver and the backEnd driver is implemented straight in hardware logic (bypassing the hypervisor with hardware support from an IOMMU and SR-IOV) as shown in **Figure 7**.
+Within the framework of the Common Telecom NFVI vision, there are three levels of documents needed to describe components and allow the practical application of the systems. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Certification**.
 
-Unlike the software only para-virtualized interfaces, this method provides better performance as it by-passes the hypervisor and unlike Direct Assignment methods, this method doesn’t require proprietary drivers to run in the workload and hence this method makes workloads portable.
+<p align="center"><img src="./figures/tech_scope.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> Scope of CNTT</p>
 
-However, this method doesn’t provide the same level of flexibility as the software only para-virtualization method as migrating workloads from one host to another is more challenging due to the hardware presence and the state it holds for the workloads using it and therefore should also not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+<a name="3.1"></a>
+## 3.1 Specification Types
```

Comment:
Feels like this could be combined with 5.1.4? Also #1269?

Reply:
5.1.4 is about networking in specific where this one is more generic. Agree more cleanup might still need to be done to make sure no replications is found.

Reply:
OK, will resolve for this PR and pick it up as a separate exercise.